### PR TITLE
also compile extension media when transpiling for code oss tests

### PR DESCRIPTION
**Pull request by joaomoreno**
Learn more: https://github.com/microsoft/vscode/issues/139672#issuecomment-1235283886
Related to: #139672

### CODE REVIEW

**Review by roblourens — DISMISSED** [review]

**Review by TylerLeonhardt — DISMISSED** [review]

**Review by TylerLeonhardt — DISMISSED** [review]

**Review by lszomoru — CHANGES_REQUESTED** [review]
1. This change seems to switch back from `transpile-client-swc` to `transpile-client`. 
2. Should this task be executed on Windows, macOS as well otherwise the platform jobs get out of sync.
3. Should this task be executed in parallel with the other tasks (ex: `yarn gulp "transpile-client" "transpile-extensions" "compile-extension-media-build")

File: build/azure-pipelines/linux/product-build-linux-client.yml [hunk 1]
```diff
@@ -218,135 +218,136 @@ steps:
...
+          VSCODE_MIXIN_PASSWORD="$(github-distro-mixin-password)" \
+            yarn gulp vscode-reh-web-linux-$(VSCODE_ARCH)-min-ci
+        displayName: Build Server
 
   - ${{ if eq(parameters.VSCODE_QUALITY, 'oss') }}:
     - script: |
         set -e
         VSCODE_MIXIN_PASSWORD="$(github-distro-mixin-password)" \
-          yarn gulp "transpile-client-swc" "transpile-extensions"
+          yarn gulp "transpile-client" "transpile-extensions"
```
Comment (joaomoreno) [inline build/azure-pipelines/linux/product-build-linux-client.yml:252]:
```suggestion
          yarn gulp "transpile-client-swc" "transpile-extensions"
```

**Comment by joaomoreno** [comment]
> * This change seems to switch back from `transpile-client-swc` to `transpile-client`.

Bad merge 🤦 

> 3. Should this task be executed in parallel with the other tasks (ex: `yarn gulp "transpile-client" "transpile-extensions" "compile-extension-media-build")

That was the [first attempt which failed in mysterious ways](https://github.com/microsoft/vscode/pull/159872/commits/abb0f7ae68edefad5e8109a351374683966af058).

> 2. Should this task be executed on Windows, macOS as well otherwise the platform jobs get out of sync.

Up to you. With this PR, I intended to point @roblourens in the right direction.

**Review by lszomoru — APPROVED** [review]

**Comment by roblourens** [comment]
This seems still stuck at `[09:44:00] Error: Command failed: /opt/hostedtoolcache/node/16.17.0/x64/bin/node /mnt/vss/_work/1/s/extensions/simple-browser/esbuild-preview.js --outputRoot /mnt/vss/_work/1/s/.build/extensions/simple-browser`

It seems like this change https://github.com/microsoft/vscode/commit/4150c81b97eac8d365f3b28063ba081f10a2e7aa worked and I have no idea what the difference would be but I could go with that one

**Comment by joaomoreno** [comment]
Closing as this was meant as a way to point to code locations.

**Pull request closed without merging**